odoo/models.py

```diff
@@ -227,6 +227,7 @@ class BaseModel(object):
 
     _name = None                # the model name
     _description = None         # the model's informal name
+    _is_business = False        # should be True for important business models
```

Comment:
Note that `sale.order.line` **is** a business model, but it is a secondary one.
The main or principal model in module `sale` is `sale.order`.
What about "principal" models?